sonar-project.properties

```diff
@@ -0,0 +1,20 @@
+sonar.projectKey=sfl:test-report-pdf
+sonar.projectName=test-report-pdf
+sonar.analysis.mode=preview
+sonar.projectVersion=1.0
+sonar.sources=.
+sonar.projectBaseDir=.
+sonar.sourceEncoding=UTF-8
+sonar.report.export.path=sonar-report.json
+# Only sonar for Python files
+sonar.language=py
+# Use this for python projects:
+sonar.python.pylint=/usr/bin/pylint3
+sonar.python.pylint_config=pylintrc
+sonar.python.pylint.reportPath=pylint-report.txt
+# Ignore solar rules S107 S3776
+sonar.issue.ignore.multicriteria=e1,e2
+sonar.issue.ignore.multicriteria.e1.ruleKey=python:S107
```

Comment:
Do we really want to ignore these rules?

Reply:
It is done by project template